tests/Makefile

```diff
@@ -1,5 +1,3 @@
-COQC=coqc -native-compiler yes
```

Comment:
This fix seems broken, as it yields:

```
$ make
success/NumberScopes.v
make: execvp: success/NumberScopes.v: Permission denied
```

could you replace this line with, e.g., `COQC=coqc` ?

Reply:
Indeed, thanks!